plugins/hls-eval-plugin/src/Ide/Plugin/Eval/CodeLens.hs

```diff
@@ -358,8 +358,9 @@ runTests EvalConfig{..} e@(_st, _) tests = do
         dbg "TEST RESULTS" rs
 
         let checkedResult = testCheck eval_cfg_diff (section, test) rs
+        let resultLines = concatMap T.lines checkedResult
```

Comment:
Honestly, I am unsure why we get `[Text]` that does not have the `show` result split into lines.

We might want to consider splitting in the `evals` function. :man_shrugging: